### DISCUSSION TITLE
ZKsync verification

### DISCUSSION THREAD

**Discussion by vpanevv** [Cyfrin/foundry-full-course-cu#3080] [Q&A]
Hi, I have problems with my first deployed smart contract on ZKsync. I can't verify it as I fixed already the suggested version. 
<img width="1288" alt="Screenshot 2024-12-20 at 14 53 04" src="https://github.com/user-attachments/assets/6db6a8bc-e0a9-4272-99b6-bbc0b5c9eb82" />

**Comment by EngrPips** [comment]
Hello @vladimirpanev95, Why didn't you deploy and verify in remix? I haven't tried verifying on `zksync etherscan` before it is difficult to be sure what is wrong.
